blocks/newslist/newslist.css

```diff
@@ -479,13 +481,217 @@
 
   .newslist .newslist-pagination-container a {
     font-family: var(--graphik-semibold);
-    font-size: 18px;
+    font-size: var(--body-font-size-s);
     font-weight: 600;
     color: #000;
     text-decoration: none;
   }
 
   .newslist .newslist-pagination-container a.current-page {
-    color: #004dff;
+    color: var(--link-color);
   }
+}
+
+/* stylelint-disable selector-class-pattern */
```

Comment:
rather than disabling the lint on whole css, can we fix these lint issues to follow best practice?

Reply:
done